### PR TITLE
C8 template edits

### PR DESCRIPTION
**Pull request by jrosen48**
This involves small edits according to the walkthrough template (see #191). I will highlight a few questions I have momentarily.

### CODE REVIEW

File: 08-wt-gradebook.Rmd [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+- linear model  
+
 ## Introduction
 
 There are a variety of data sources to explore in the education field. Student assessment scores can be examined for progress towards goals. The text from a teacher’s written classroom observation notes about a particular learner’s in-class behavior or emotional status can be analyzed for trends. We can tap into the exportable data available from common learning software or platforms popular in the K-12 education space.
 
-## Data Sources
+### Background
+
+This walkthrough goes through a series of analyses using the data science framework. The first analysis centers around a ubiquitous K-12 classroom tool: the gradebook. While gradebook data is nearly ubiquitious in education, it is sometimes ignored in favor of data collected by evaluators and researchers or data from state-wide tests. Nevertheless, it represents an important, occasionally untapped data source, and one for which a data science approach can reveal the potential of the data science framework for analyzing a range of educational data sources.
```
Comment (jrosen48) [inline 08-wt-gradebook.Rmd]: is this an okay addition?

File: 08-wt-gradebook.Rmd [hunk 2]
```diff
@@ -64,7 +82,9 @@ gradebook <- ExcelGradeBook
 
 Your environment will now have two versions of the dataset. There is *ExcelGradeBook* which was the original dataset imported. There is also *gradebook* which is currently a copy of *ExcelGradeBook*. As you progress through this section, we will work primarily with the *gradebook* dataset. Additionally, while working onward in this section of the book, if you make a mistake and mess up the *gradebook* data frame and are not able to fix it, you can reset the data frame to return to the same state as the original *ExcelGradeBook* data frame by running `gradebook <- ExcelGradeBook` again. This will overwrite any errors in the *gradebook* data frame with the originally imported *ExcelGradeBook* data frame. Afterwards, just continue running code from this point in the text.
 
-## Tidy Data
+## Process data
```
Comment (jrosen48) [inline 08-wt-gradebook.Rmd]: does the distinction beneath this heading between `### Tidy Data` and `### Create new variables and further process the data` make sense?

File: 08-wt-gradebook.Rmd [hunk 3]
```diff
@@ -1,12 +1,30 @@
 # Walkthrough 2: Approaching Gradebook Data From a Data Science Perspective {#c08}
 
+## Vocabulary
```
Comment (jrosen48) [inline 08-wt-gradebook.Rmd]: are these on the right track?

File: 08-wt-gradebook.Rmd [hunk 4]
```diff
@@ -116,7 +136,7 @@ At last, the formerly untidy Excel sheet has been turned into a useful data fram
 view(gradebook)
 ```
 
-## Process Data
+### Create new variables and further process the data
```
Comment (jrosen48) [inline 08-wt-gradebook.Rmd]: just to distinguish this from `## Process data` above

File: 08-wt-gradebook.Rmd [hunk 5]
```diff
@@ -154,7 +174,9 @@ View the new data frame and note which columns were selected for this new data f
 view(classwork_df)
 ```
 
-## Visualize Data
+## Analysis and results
```
Comment (jrosen48) [inline 08-wt-gradebook.Rmd]: okay with this heading for the subsequent sections (for which I indexed the headings "down" one level, i.e., level 2 headings become level 3)?

**Comment by jrosen48** [comment]
some fairly minor edits to what is a solid chapter - @kierisi would you be willing to take a look at and review these?

File: 08-wt-gradebook.Rmd (hunk 1 above)
Comment (kierisi) [reply 08-wt-gradebook.Rmd]: sorry for the late reply - GH notifications are getting split between spam and my inbox. this is good! I like that it highlights the ubiquitousness of gradebooks as an untapped resource!

**Comment by kierisi** [comment]
I'm good with this being merged, but wanted to leave it so you don't lose the comments :)

**Comment by jrosen48** [comment]
Thanks much! Working on now.

File: 08-wt-gradebook.Rmd (hunk 1 above)
Comment (jrosen48) [reply 08-wt-gradebook.Rmd]: sweet

**Comment by jrosen48** [comment]
Thank you @kierisi for reviewing and approving this

**Pull request merged**